pyblazing/pyblazing/apiv2/context.py

```diff
@@ -1309,11 +1309,11 @@ def __init__(
         # want to use config_options and not self.config_options
         # since its not encoded
         if "BLAZING_LOGGING_DIRECTORY" in self.config_options:
```

Comment:
Due to load_config_options_from_env function returns the encoded config_options,
the variable name should be encoded too, otherwise it will have no effect:

For example:
`if "BLAZING_LOGGING_DIRECTORY".encode() in self.config_options:`

Reply:
good call.